lib/plugins/aws/deployFunction/index.js

```diff
@@ -66,6 +72,38 @@ class AwsDeployFunction {
     });
   }
 
+  updateFunctionConfiguration() {
+    const params = omitEmpty({
```

Comment:
Does this include config which is defined on a the `provider` config as well?

See: https://serverless.com/framework/docs/providers/aws/guide/serverless.yml/

Reply:
Good catch!